client/components/payment-methods-checkboxes/payment-method-checkbox.scss

```diff
@@ -43,6 +43,21 @@
 			margin: 0;
 			color: $gray-600;
 		}
+
+		// To be removed when @wordpress/components is upgraded to 17.0.0.
+		.components-visually-hidden {
```

Comment:
Apparently only this lines seem to affect the visual

```
    height: 1px;
    overflow: hidden;
    position: absolute;
```

Reply:
Thanks for pointing that. Yep, I had noticed it, but because this relies on external styles, I opted for adding the [original set of rules](https://github.com/WordPress/WordPress/blob/master/wp-includes/css/dist/components/style.css#L5042) that were being overwritten, which are also the ones included in the [latest version](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/visually-hidden/styles.js) of the component. All good with updating it if the other reviewer agrees as well, though.